currency_rate_update/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Currency Rate Update',
-    'version': '12.0.1.1.2',
```

Comment:
This is not a patch release, it's a full-scale minor change.

Reply:
12.0.1.2.1 ?

Reply:
`12.0.1.2.0`, you can even omit this change - OCA bot will take care of it during merge

Reply:
Why .1 in the end?